app/controllers/activitypub/outboxes_controller.rb

```diff
@@ -57,9 +57,8 @@ def prev_page
   def set_statuses
     return unless page_requested?
 
-    @statuses = @account.statuses.permitted_for(@account, signed_request_account)
     @statuses = cache_collection_paginated_by_id(
-      @statuses,
+      @account.statuses.permitted_for(@account, signed_request_account),
```

Comment:
Not a big deal, but this still lists toots of suspended accounts, doesn't it?

Reply:
I'd have to check but don't we still return HTTP 410 for ActivityPub routes belonging to a suspended account?

Reply:
Hm, yeah, it's using `AccountOwnedConcern` which still returns 410 if the account is suspended. So that's fine for this PR.